README.md

```diff
@@ -154,14 +160,14 @@ as an argument to the console (either as an `<IP>:<PORT>` or
 `<DNS_NAME>:<PORT>`). for example:
 
 ```shell
-$ cargo run -- http://my.great.console.app.local:5555
+cargo run -- http://my.great.console.app.local:5555
 ```
 
 the console command-line tool supports a number of additional flags to configure
 its behavior. the `-h` or `--help` flag will print a list of supported
 command-line flags and arguments:
 
-```
+```sh
```

Comment:
i don't think this should be treated as a shell script. we should probably just treat it as plain text:
```suggestion
```text
```